#! /bin/sh -e

# DP: Add some extra UTF-8 locales. Borrowed patch from SuSe.

if [ $# -ne 2 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
    -patch) patch -d "$2" -f --no-backup-if-mismatch -p0 < $0;;
    -unpatch) patch -d "$2" -f --no-backup-if-mismatch -R -p0 < $0;;
    *)
	echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
	exit 1
esac
exit 0

--- localedata/SUPPORTED~	2001/02/14 01:37:11
+++ localedata/SUPPORTED	2001/06/15 04:30:23
@@ -32,15 +32,18 @@
 de_BE@euro/ISO-8859-15 \
 de_CH/ISO-8859-1 \
 de_DE/ISO-8859-1 \
+de_DE.UTF-8/UTF-8 \
 de_DE@euro/ISO-8859-15 \
 de_LU/ISO-8859-1 \
 de_LU@euro/ISO-8859-15 \
 el_GR/ISO-8859-7 \
+el_GR.UTF-8/UTF-8 \
 en_AU/ISO-8859-1 \
 en_BW/ISO-8859-1 \
 en_CA/ISO-8859-1 \
 en_DK/ISO-8859-1 \
 en_GB/ISO-8859-1 \
+en_GB.UTF-8/UTF-8 \
 en_HK/ISO-8859-1 \
 en_IE/ISO-8859-1 \
 en_IE@euro/ISO-8859-15 \
@@ -49,6 +52,7 @@
 en_PH/ISO-8859-1 \
 en_SG/ISO-8859-1 \
 en_US/ISO-8859-1 \
+en_US.UTF-8/UTF-8 \
 en_ZA/ISO-8859-1 \
 en_ZW/ISO-8859-1 \
 es_AR/ISO-8859-1 \
@@ -75,7 +79,7 @@
 et_EE/ISO-8859-1 \
 eu_ES/ISO-8859-1 \
 eu_ES@euro/ISO-8859-15 \
-fa_IR/UTF-8 \
+fa_IR.UTF-8/UTF-8 \
 fi_FI/ISO-8859-1 \
 fi_FI@euro/ISO-8859-15 \
 fo_FO/ISO-8859-1 \
@@ -84,6 +88,7 @@
 fr_CA/ISO-8859-1 \
 fr_CH/ISO-8859-1 \
 fr_FR/ISO-8859-1 \
+fr_FR.UTF-8/UTF-8 \
 fr_FR@euro/ISO-8859-15 \
 fr_LU/ISO-8859-1 \
 fr_LU@euro/ISO-8859-15 \
@@ -93,7 +98,7 @@
 gl_ES@euro/ISO-8859-15 \
 gv_GB/ISO-8859-1 \
 he_IL/ISO-8859-8 \
-hi_IN/UTF-8 \
+hi_IN.UTF-8/UTF-8 \
 hr_HR/ISO-8859-2 \
 hu_HU/ISO-8859-2 \
 id_ID/ISO-8859-1 \
@@ -103,6 +108,7 @@
 it_IT@euro/ISO-8859-15 \
 iw_IL/ISO-8859-8 \
 ja_JP.EUC-JP/EUC-JP \
+ja_JP.UTF-8/UTF-8 \
 ka_GE/GEORGIAN-PS \
 kl_GL/ISO-8859-1 \
 ko_KR.EUC-KR/EUC-KR \
@@ -110,7 +116,7 @@
 lt_LT/ISO-8859-13 \
 lv_LV/ISO-8859-13 \
 mk_MK/ISO-8859-5 \
-mr_IN/UTF-8 \
+mr_IN.UTF-8/UTF-8 \
 ms_MY/ISO-8859-1 \
 mt_MT/ISO-8859-3 \
 nl_BE/ISO-8859-1 \
@@ -125,6 +131,7 @@
 pt_PT@euro/ISO-8859-15 \
 ro_RO/ISO-8859-2 \
 ru_RU/ISO-8859-5 \
+ru_RU.UTF-8/UTF-8 \
 ru_RU.KOI8-R/KOI8-R \
 ru_UA/KOI8-U \
 sk_SK/ISO-8859-2 \
@@ -140,14 +147,17 @@
 uk_UA/KOI8-U \
 ur_PK/UTF-8 \
 uz_UZ/ISO-8859-1 \
-vi_VN/UTF-8 \
+vi_VN.UTF-8/UTF-8 \
 vi_VN.TCVN/TCVN5712-1 \
 wa_BE/ISO-8859-1 \
 wa_BE@euro/ISO-8859-15 \
 yi_US/CP1255 \
 zh_CN/GB2312 \
 zh_CN.GB18030/GB18030 \
 zh_CN.GBK/GBK \
+zh_CN.UTF-8/UTF-8 \
 zh_HK/BIG5-HKSCS \
+zh_HK.UTF-8/UTF-8 \
 zh_TW/BIG5 \
 zh_TW.EUC-TW/EUC-TW \
+zh_TW.UTF-8/UTF-8 \
